docs/interactions/application-commands/slash-commands.mdx

```diff
@@ -35,8 +35,8 @@ import discord
 
 bot = discord.Bot(debug_guilds=[...]) # specify the guild IDs in debug_guilds
 
-# since global slash commands can take up to an hour to register,
 # we need to limit the guilds for testing purposes
+# so only you see it, and not everyone else
```

Comment:
We set debug_guilds here so you can test your commands without other users accidentally having access to commands which aren't ready for production 


Or similar

The actual wording sounds wrong